### DISCUSSION TITLE
Syndicate Business Mission Series: Inefficient Passive Income & Syndicate Flavor

### DISCUSSION THREAD

**Discussion by reticent-rem** [endless-sky/endless-sky#10552] [Show and tell]
## Preface
Hello! Previously this was a part of one of the PRs, but this is more about the general concept and feedback for the mission series as a whole so I've split it off as a discussion. Thank you to everyone who's contributed and given constructive feedback so far (Anarchist2, EjoThims, Hecter94, mOctave, Pointedstick, roadrunner56, TheGiraffe3, tibetiroka, and anyone else I've missed!)

## Summary
This a proposed 4 part stand-alone mission series in Syndicate space with the 2 goals of:

1. Fleshing out being an investor in the Syndicate via the player being involved with a Syndicate subsidiary business (Megaparsec)
2. Creating a purposefully inefficient way to convert credits into a cashflow stream, with specific detail in the Finance Math section.

At each mission set, I am planning to create more developed branches at each level where the player has the option to ruthlessly sacrifice the environment and human rights to maximize their returns, or take a lower financial return but getting more intangible rewards of thankful people and a noticeable improvement in worker and environmental conditions in Syndicate space.

I also wanted the follow the [StyleGoals](https://github.com/endless-sky/endless-sky/wiki/StyleGoals) of not having black and white factions by portraying at mid-level Syndicate as generally nice but with fewer scruples than most people, and also by getting the player to make decisions about personal gain vs moral good.

**Overall blueprint for this mission chain:**

1. **1M investment**: simpler mission introducing the player to Megaparsec Incorporated and Howard Vendeur, an ambitious salesman (#10393 done and merged!)
2. **10M investment**: Howard asks the player for a favor in capturing a Marauder ship and the player can choose between 3 investments that are good, neutral, and terrible for society, with returns to match (#10422 done and merged!)
   * Two culture missions, one where the player gets a financial report of their investment causing a dangerous accident or improved morale depending on their investment decision, and one where the player can experience the human impact of their choice on Syndicate workers (#10562 done and merged!)
3. **100M investment**: Howard asks the player for another favor which results in another investment choice with good, neutral, and terrible impacts for society, with returns to match and a larger impact by building a testing facility on Crossroads (#11298 done and ready for review!)
   * Another culture mission where the player overhears people celebrating great times or bemoaning the sad state of affairs depending on their investment decision.
   * A more permanent change reflecting the larger investment and impact, either changing a line on the planet description or adding some news/hails mentioning the great / awful things happening on the planet.
   * ~I'm kicking around the idea of staying on the Marauder track and having one of the players investment choices unlocking a Marauder Quicksilver at advanced Megaparsec shipyards, but that's an open thought. The player is only offered this mission when they have 300M credits, so it wouldn't be unlocking anything too exciting at that level.~
4. **1B investment**: Howard asks the player for a final favor which results in another investment choice with good, neutral, and terrible impacts for society, with returns to match and large enough to have (minor) repercussions on everywhere Megaparsec operates.
   * A final culture mission where people thank the player or spit in their face depending on their investment decision.
   * A more permanent change reflecting the even larger investment and impact, either changing a line on the planet description or adding some news/hails mentioning the great / awful things happening across multiple planets.
   * ~If the the Marauder idea stays, one of the players investment choices could be unlocking the rest of Megaparsec's ships (Manta, Splinter, and Bounder) as Marauder variants at advanced Megaparsec shipyards.~
5. **Epilogue / Easter Egg**: If the original pirate from the first mission (Golden Jewel) survives every encounter with First Last through the 1B investment, I'm adding a short mission where they ask the player for a job and the player gets a maintenance free Vanguard (paid for on the company dime).

<details>
<summary>Finance Math</summary>
<br>

This is the broad concept I am envisioning, but would love feedback on if the returns feel too low or too high to people.

Current plan has 1 choice for the first mission, and then 3 options for following missions that are kinder with lower returns or greedy with higher returns. For each choice, the first number is the amount of credits per day received, (and the number in parenthesis is how many years it takes before you're no longer losing money).

So for the Kind option in the 10M Credits mission, First Last receives 2,000 credits per day, and it takes 13.7 years to go from -10M credits back to 0. This can be even longer in some circumstances, like if First Last was mean to Howard and has to pay back 11M instead of 10M.

  | 1M Credits | 10M Credits | 100M Credits | 1B Credits
-- | -- | -- | -- | --
Kind |   | 4,000 (6.8) | 20,400 (13.4 years) | 51,300 (53.4)
Normal | 1,000 (2.7 years to stop losing money) | 5,400 (5.1) | 27,400 (10 years) | 68,500 (40.0)
Greedy |   | 8,200 (3.3) | 41,000 (6.8 years | 102,700 (26.7)

 For each investment after the first, Howard also arranges for a commercial loan for First Last. The below table lists out the terms of the loan by term, rate, daily payment, and total paid over the loan. The daily interest rate is extraordinarily low compared to the normal loans First Last can get, but this is a trap. 
 
 Over the life of the loan First Last pays 3 times as much compared to paying outright, which means that **using the loan takes 3 times as long for the investment to break even**. Even if First Last paid off parts of the loan, the loan's length and ES's loan mechanics means that any payments will be mostly interest until the last few years of a loan for the ones here.

Looking back at the Kind option for 100M credits, not paying off the loan means it will take ~40 years to break even instead of ~13.4 years. And of the 42,282 daily loan payment, 40,000 of that is initially going just to interest. It takes over 15 years of the 20 year loan before there's more principal being paid than interest. And that payoff is even worse if the player was Mean to Howard, with a higher investment cost and loan rate.

  | 1M Credits | 10M Credits | 100M Credits | 1B Credits
-- | -- | -- | -- | --
Loan term (years) | N/A | 10 | 20 | 40
Loan daily rate |   | 0.08% | 0.04% | 0.02%
Daily Payment |   | 8,457 | 42,282 | 211,405
Total Paid |   | 30,866,699 | 308,656,717 | 3,086,515,786

</details>

## Questions for You!

1. Does the financial math feel reasonable to where it's a bad way to make money, but you could still see people making the investments if they're lazy and/or plan to play over decades?
2. Is it okay to have one of the investment options at 100M & 1B unlock some Megaparsec Marauder ships to be sold at advanced Megaparsec shipyards, where the investment missions only occur when the player has 300M & 2B credits (I won't do this if it's controversial, but definitely want that figured out ahead of time).
   - ~I'm hoping for a decision here with either someone saying they don't want this or a soft approval from a dev that this concept won't be vetoed later on.~ Marauder ships aren't meant to be produced and sold in regular shipyards.
3. Any other feedback on the mission series as a whole or on ideas not yet in PR's.

**Comment by TheGiraffe3** [comment]
I said this on Part 2, but I'll say it again:
I think this is a really great idea!
The financial math seems fine to me - and we need more ways of making passive income.
I think it's fine if some lower-tech Marauders are sold - after all, they can't be _too_ hard to build, and when you have 300M Marauders aren't too big a balance problem.

**Comment by roadrunner56** [comment]
Sorry it took me so long to actually comment on this. I really like the idea of this mission and what I've seen of your writing so far. I think it definitely offers more insight into the kinds of people within the Syndicate, and how their (and your!) actions have a direct effect on the state of the galaxy.
The way the math works out on these investments seems reasonable on first glance, if a little harsh to the player. 
With regards to the marauder ships, they're more intended as aftermarket modifications that pirates make to their ships rather than ships that get produced and sold in regular shipyards. They were meant to be an exclusive reward for players who managed to capture them, so I'd avoid having them be sold in Megaparsec shipyards, which kind of undercuts the uniqueness of that reward.
I'd also avoid adding new ship models distinct from marauder ships as a replacement reward. Since there are already a lot of human ships in ES, we want to limit the amount of new human ships added to the game. 
Perhaps you could give the player a discount at Megaparsec shipyards instead?

**Reply by reticent-rem** [reply]
Thanks for the feedback, that's very kind! I'm totally happy excluding marauder ships or any new ship models as rewards. I like the idea of a Megaparsec discount, though if that ends up not feasible for whatever reason I think the series still works well with no special rewards.

On the math, it's good that it seems reasonable, although I've now gotten a few comments that it's perhaps a bit too harsh to the player. Would there be any concern if I doubled the returns in the first 2 missions? It would still take multiple years before the player stops losing money on net, and in a steady state they'd be making enough to buy:

1. At 1,000 credits per day: 1 free S3 Thermionic Generator per year (360,000 credits) and change (5,000 credits)
2. At 8,200 per day: Just under 2 free Fission Reactors per year (2X 1.5M credits)


  | 1M Credits | 10M Credits 
-- | -- | -- 
Kind |   | 4,000 (6.8)
Normal | 1,000 (2.7 years to stop losing money) | 5,400 (5.1) 
Greedy |   | 8,200 (3.3)

**Reply by roadrunner56** [reply]
I think that still works, yeah.

**Reply by reticent-rem** [reply]
Okay, just updated both!
